scripts/deployments/validate_simcore_stack_yml.bash

```diff
@@ -17,6 +17,32 @@ chmod +x yq
 _yq=$(realpath ./yq)
 export _yq
 #
+# Check if the Docker Compose file has a top-level secrets section
+if $_yq e ".secrets" ${COMPOSE_FILE} > /dev/null; then
```

Comment:
_Idea_ :bulb:

It could be handy to keep some static dummy secrets files to use for validation